vignettes/algorithm.Rmd

```diff
@@ -238,6 +238,7 @@ where $l_{i,jj}$ are the diagonal entries of the factor $L_i$ and we have used t
 - the determinant of the product of matrices is the product of the determinants,
 - the determinant of the transposed matrix is the same as the original one,
 - the determinant of a triangular matrix is the product of the diagonal.
+When weights $Z_i$ is specified, we have $\frac{1}{2}\log\det(\Omega) = \sum_{i=1}^{n}\sum_{j=1}^{m_i}\log(z_{i, j} l_{i, jj})$, where $z_{i, j}$ are the diagonal entries of the weight $Z_i$.
```

Comment:
similar here - not sure if we need this? e.g. if we explain above already how to calculate $L_i$ then this contains already everything needed here?